pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -48,6 +52,18 @@ const (
 	DefaultCNIDir     = "/opt/cni/bin"
 
 	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
+
+	// the hostport chain
+	kubenetHostportChain utiliptables.Chain = "KUBENET-HOSTPORT"
+	// the kubenet postrouting chain
+	kubenetPostroutingChain utiliptables.Chain = "KUBENET-POSTROUTING"
+	// the mark-for-masquerade chain
+	kubenetMarkChain utiliptables.Chain = "KUBENET-MARK"
+	// TODO: do not hard code mark
```

Comment:
How do we resolve this TODO?  do we need a flag to kubelet?  Why is this a different bit than the kube-proxy? What is the long-term plan to manage these